CMakeLists.txt

```diff
@@ -273,7 +273,7 @@ if(ENABLE_PROGRAMS)
     add_subdirectory(programs)
 endif()
 
-ADD_CUSTOM_TARGET(apidoc
+ADD_CUSTOM_TARGET(${target_prefix}apidoc${target_suffix}
```

Comment:
So far we discussed about a prefix but not a suffix. Do you really need a suffix as well?
Otherwise as target_prefix is an Mbed TLS CMake global variable better to prefix it itself with "mbedtls" thus
"mbedtls_target_prefix" instead "target_prefix".

Reply:
Done.